backend/app/src/test/kotlin/io/tolgee/api/v2/controllers/v2ImageUploadController/SecuredV2ImageUploadControllerTest.kt

```diff
@@ -48,9 +43,9 @@ class SecuredV2ImageUploadControllerTest : AbstractV2ImageUploadControllerTest()
       mapOf("fileName" to image.filenameWithExtension)
     )
 
-    currentDateProvider.move(Duration.ofSeconds(10))
+    moveCurrentDate(Duration.ofSeconds(10))
     performAuthGet("/uploaded-images/${image.filename}.jpg?token=$token").andIsUnauthorized
-    currentDateProvider.forcedDate = null
+    clearForcedDate()
```

Comment:
To it in afterEach. This won't be executed when the test fails and so it would break the other tests.